### PR TITLE
#169258571 Add sass preprocessing

### PR DESCRIPTION
**Pull request by NiyongaboEric**
#### What does this PR do?
We will be using SASS preprocessing to style our project.

#### Description of Task to be completed?
- Update Webpack 
- Add `node-sass` and `sass-loader` npm packages
- Add Eslint rule to accept both .jsx and .js files extensions

#### How should this be manually tested?
- `git clone ch-setup-SASS-169258571`
- `cd team-odd-bn-frontend`
- `git checkout ch-setup-SASS-169258571`
- `npm install` packages
- `npm run dev` 
- Open `App.scss`, try to change background color by editing `$body-color: #red` the whole body will change.

#### Any background context you want to provide?
We are going to have two different ways to style pages. Pure CSS and SCSS preprocessor.

#### What are the relevant pivotal tracker stories?
[#169258571](https://www.pivotaltracker.com/story/show/169258571)

#### Screenshots (if appropriate)
Use `App.scss` file to change background color of body
Screen Shot 2019-12-24 at 18 22 34

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@ dist
 package-lock.json
 .DS_Store
 config/
+coverage/
```
Comment (hezronkimutai) [inline .gitignore]: Eol here @NiyoEric

**Review by hezronkimutai — CHANGES_REQUESTED** [review]
Nice work Eric, just check the comment I left.

**Review by hezronkimutai — APPROVED** [review]
nice job.

**Review by victkarangwa — APPROVED** [review]
Good job!

**Review by nziokaivy — APPROVED** [review]
Great job @NiyongaboEric !

This is good to go!

**Review by shemaeric — APPROVED** [review]
Good work @NiyongaboEric

**Pull request merged**